.npmignore

```diff
@@ -1 +0,0 @@
-can_publish
```

Comment:
hmm, nothing to npmignore?

Reply:
Not sure I see the point in maintaining an additional ignore file for no gain. If anything we should be using `package.json#files`.